Documentation/Getting-Started/glossary.md

```diff
@@ -0,0 +1,87 @@
+# Glossary
+
+### Ceph
+
+Ceph is a distributed network storage and file system with distributed metadata management and POSIX semantics.
+
+### MON
+
+Ceph Monitor is a daemon that maintains a map of the state of the cluster. This “cluster state” includes the monitor map, the manager map, the OSD map, and the CRUSH map. A Ceph cluster must contain a minimum of three running monitors in order to be both redundant and highly-available. Ceph monitors and the nodes on which they run are often referred to as “mon”s.
+
+### OSD
+
+Ceph Object Storage Daemon. The Ceph OSD software, which interacts with logical disks.
+
+### MGR 
+
+The Ceph manager software, which collects all the state from the whole cluster in one place.
+
+### RADOS
+
+Reliable Autonomic Distributed Object Store. RADOS is the object store that provides a scalable service for variably-sized objects. The RADOS object store is the core component of a Ceph cluster.
+
+### RADOS Cluster
+
+A proper subset of the Ceph Cluster consisting of OSDs, Ceph Monitors, and Ceph Managers.
+
+### MDS
+
+The Ceph MetaData Server daemon. Also referred to as “ceph-mds”. The Ceph metadata server daemon must be running in any Ceph cluster that runs the CephFS file system. The MDS stores all filesystem metadata.
+
+### RBD
+
+The block storage component of Ceph. Also called "RADOS Block Device". Its a software instrument that orchestrates the storage of block-based data in Ceph.
+
+### RGW
+
+RADOS Gate Way. The component of Ceph that provides a gateway to both the Amazon S3 RESTful API and the OpenStack Swift API.
+
+For more Ceph related terms, please refer to the [Ceph Glossary page](https://docs.ceph.com/en/latest/glossary/).
+
+### Kubernetes
+
+    Kubernetes, also known as K8s, is an open-source system for automating deployment, scaling, and management of containerized applications.
```

Comment:
```suggestion
Kubernetes, also known as K8s, is an open-source system for automating deployment, scaling, and management of containerized applications.
```

Reply:
sure it makes sense. I have increased the Heading size of Both Ceph and Kubernetes.